src/components/GettingStarted.tsx

```diff
@@ -1,31 +1,94 @@
   return (
-    <div className="not-prose my-16 xl:max-w-none">
-      <Link
-        href={'/guides/getting-started'}
-        className="group relative flex gap-8 rounded-2xl bg-zinc-50 p-6 transition-shadow hover:shadow-md hover:shadow-zinc-900/5 dark:bg-white/2.5 dark:hover:shadow-black/5"
-      >
-        <div className="absolute inset-0 rounded-2xl ring-1 ring-inset ring-zinc-900/7.5 group-hover:ring-zinc-900/10 dark:ring-white/10 dark:group-hover:ring-white/20" />
-        <div className="flex h-20 w-20 items-center justify-center rounded-full border-8 border-secondary-900/40 bg-secondary-200 dark:border-secondary-900/60 dark:bg-secondary-300">
-          <PlayIcon className="h-8 w-8 text-white" />
+    <div
+      key={block.href}
+      onMouseMove={onMouseMove}
+      className={
+        'group relative flex rounded-2xl bg-zinc-50 transition-shadow hover:shadow-md hover:shadow-zinc-900/5 dark:bg-white/2.5 dark:hover:shadow-black/5'
+      }
+    >
+      <ResourcePattern {...block.pattern} mouseX={mouseX} mouseY={mouseY} />
+      <div className="absolute inset-0 rounded-2xl ring-1 ring-inset ring-zinc-900/7.5 group-hover:ring-zinc-900/10 dark:ring-white/10 dark:group-hover:ring-white/20" />
+      <div className="relative w-full rounded-2xl p-8">
+        <div
+          className={clsx(
+            'flex items-center gap-2.5',
+            block.description && 'mb-4'
+          )}
+        >
+          {block.icon && (
+            <block.icon
+              className={cn('h-6 w-6 text-primary-light', block.iconClassName)}
+            />
+          )}
+          <h3 className="text-lg font-semibold leading-7 text-zinc-900 dark:text-white">
+            <Link href={block.href}>
+              <span className="absolute inset-0 rounded-2xl" />
+              {block.name}
+            </Link>
+          </h3>
         </div>
-        <div className="w-7/12 lg:w-auto">
-          <h2 className="text-xl font-semibold text-zinc-900 dark:text-white">
-            Getting Started
-          </h2>
-          <p className="mt-1 max-w-md text-sm text-zinc-600 dark:text-zinc-400">
-            Get started with Nitric and unleash the power of rapid cloud and
-            serverless application development in just a few simple steps.
+        {block.description && (
+          <p className="mt-1 text-base text-zinc-600 dark:text-zinc-400">
+            {block.description}
           </p>
-        </div>
-      </Link>
+        )}
+      </div>
+    </div>
+  )
+}
+
+const gettingStartedBlocks = [
+  {
+    href: '/concepts/introduction',
+    name: 'What is Nitric',
+    icon: InformationCircleIcon,
+    description:
+      'Learn about the Nitric architecture and how it works under the hood.',
+  },
+  {
+    href: '/getting-started/quickstart',
+    name: 'Get started with Nitric',
+    icon: RocketLaunchIcon,
+    description: 'Start deploying in 15-minutes with our Quick Start guide.',
+  },
+  {
+    href: '/concepts/extending-nitric',
```

Comment:
For now, can we link this to /reference/providers/custom/building-custom-provider? We can come back later and create new content for the concepts section as part of another story.